jupyros/ros2/key_input.py

```diff
@@ -0,0 +1,94 @@
+## Keyboard input for Jupyter Ros 2
+"""
+Author:         Luigi Dania
+Email:          Luigi@dobots.nl
+Github:         https://github.com/ldania
+
+Company:        Dobots
+Company Repo:   https://github.com/dobots/ 
+
+
+"""
+
+import sys
+sys.path.append("./../jupyter-ros")
+import jupyros.ros2 as jr2
+from ipyevents import Event 
+from ipywidgets import Output
+from ipycanvas import Canvas
+import rclpy
+
+
+
+
+
+class key_input:
```

Comment:
This seems to be working so far. It detects the keyboard inputs and shows them on the notebook which is great. I think what is being displayed could be improved upon a little more. Currently, I see a gray square and farther at the bottom is where the keyboard inputs are being printed. 

![image](https://user-images.githubusercontent.com/23283243/185391611-78bc8ae7-0289-4cbc-8a08-0a1b1c487a29.png)

One idea would be to display arrows on the canvas and when the keyboard arrows are being pressed, those could be highlighted on the canvas. This should be simple to do with `ipycanvas`. 
Or if what's important is simply the inputs and it's not necessary to have an animation, then simply printing the strings should be sufficient.